pkg/cmd/logs.go

```diff
@@ -382,9 +385,15 @@ func logsControllerManager() {
 }
 
 // logsVpnSeed prints the logfile of the vpn-seed container
-func logsVpnSeed(shootName string) {
+func logsVpnSeed(shootTechnicalID string) {
 	fmt.Println("-----------------------Kube-Apiserver")
-	logPodSeed("kube-apiserver", shootName, "vpn-seed")
+	if shootTechnicalID == emptyString {
+		output, err := ExecCmdReturnOutput("kubectl", "config", "view", "--minify", "--output", "jsonpath={..namespace}")
+		checkError(err)
+		logPodSeed("kube-apiserver", output, "vpn-seed")
+	} else {
+		logPodSeed("kube-apiserver", shootTechnicalID, "vpn-seed")
+	}
```

Comment:
```suggestion
	var err error
	if shootTechnicalID == emptyString {
		shootTechnicalID, err = ExecCmdReturnOutput("kubectl", "config", "view", "--minify", "--output", "jsonpath={..namespace}")
		checkError(err)
	}
	logPodSeed("kube-apiserver", shootTechnicalID, "vpn-seed")
```

however, instead of reading the namespace from the current context, can't you make use of `GetFromTargetInfo(targetReader, "shootTechnicalID")` to get the technicalID?

Reply:
@petersutter Great suggestion but from some quick test seems like the `GetFromTargetInfo()` method only work on the shoot and return the following error when executed on the seed 
![image](https://user-images.githubusercontent.com/11049349/98738591-f1cfb800-2375-11eb-8f8a-649c21099b71.png)

the followiing code seems to be the good way to go about it for now :

Reply:
ah ok, yes this won't work then

Reply:
the suggested change is still valid tho (independent of GetFromTargetInfo)

Reply:
sorry I just curious how can I reproduce it? I may need to check whether is any defect in the 'getFromTargetInfo' method and fix it.

however when I pull this PR and test it. it seems good. maybe I miss something in your scenario.  Please let me know, thanks 


![image](https://user-images.githubusercontent.com/42234376/99043889-da890880-25c9-11eb-99d7-4cb258fd3a47.png)

![image](https://user-images.githubusercontent.com/42234376/99043973-fe4c4e80-25c9-11eb-88d0-2d7f2998414b.png)

![image](https://user-images.githubusercontent.com/42234376/99044031-145a0f00-25ca-11eb-9fd8-f26be9172842.png)

Reply:
@tedteng You were right I was setting my environment the wrong way. It's working now ! Thx